### PR TITLE
Update Travis CI matrix

### PR DESCRIPTION
**Pull request by tjarratt**
Dropping support for iOS 7 for now. Would like to add that back once this goes green.

### CODE REVIEW

**Comment by briancroom** [comment]
I had to restart one of the jobs (sad! I was hoping that moving to the Xcode 7.2 image would resolve some of those issues), but this did turn green now!

**Comment by tjarratt** [comment]
The Xcode 7.2 image is ... Nicer in some ways but it introduces more asynchronous behavior in starting the simulator. 

I regret waiting so many months before trying to get these tests running in a modern Xcode on Travis. 

Sent From A Very Small Keyboard

File: Rakefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 CONFIGURATION = "Release"
 BUILD_SDK_VERSION = ENV['BUILD_SDK_VERSION'] || ""
-SIMULATOR_VERSIONS = ENV['SIMULATOR_VERSIONS'] || "9.0"
+SIMULATOR_VERSIONS = ENV['SIMULATOR_VERSIONS'] || "9.2"
```
Comment (akitchen) [inline Rakefile]: I would love for us to auto-magically discover this value.

File: UIKit/Spec/Stubs/UINavigationControllerSpec+Spec.mm [hunk 2]
```diff
@@ -24,11 +24,16 @@ @implementation CustomToolbar
...
     bool has = false;
-#ifndef __IPHONE_9_0
+
+    NSOperatingSystemVersion version = [[NSProcessInfo processInfo] operatingSystemVersion];
+    if (version.majorVersion >= 9) {
+        return false;
+    }
+
 #if __LP64__
     has = true;
```
Comment (akitchen) [inline UIKit/Spec/Stubs/UINavigationControllerSpec+Spec.mm]: we need to confirm these issues still exist.  Has anyone looked at it recently?

**Comment by akitchen** [comment]
I've added a few comments, but nothing new.  

Does someone feel like fixing up the merge issues so we can merge this?

Thanks @tjarratt !!

File: UIKit/Spec/Stubs/UINavigationControllerSpec+Spec.mm (hunk 2 above)
Comment (tjarratt) [inline UIKit/Spec/Stubs/UINavigationControllerSpec+Spec.mm]: I looked at that issue. Confirmed it still exists. Rewrote that code so it
would continue to not cause the test suite to fail.

On Monday, February 8, 2016, Andrew Kitchen <notifications@github.com>
wrote:

> In UIKit/Spec/Stubs/UINavigationControllerSpec+Spec.mm
> <https://github.com/pivotal/PivotalCoreKit/pull/180#discussion_r52218365>:
>
> > @@ -24,11 +24,16 @@ @implementation CustomToolbar
> >   */
> >  bool hasAssertBug() {
> >      bool has = false;
> > -#ifndef __IPHONE_9_0
> > +
> > +    NSOperatingSystemVersion version = [[NSProcessInfo processInfo] operatingSystemVersion];
> > +    if (version.majorVersion >= 9) {
> > +        return false;
> > +    }
> > +
> >  #if __LP64__
> >      has = true;
>
> we need to confirm these issues still exist. Has anyone looked at it
> recently?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pivotal/PivotalCoreKit/pull/180/files#r52218365>.
>


-- 
Magnetized From My Etch-A-Sketch

**Pull request merged**